inc/classes/admin/settings/class-page.php

```diff
@@ -888,21 +888,46 @@ private function media_section() {
 			]
 		);
 
-		$rocket_maybe_disable_lazyload_plugins = [];
+		$disable_images_lazyload  = [];
+		$disable_iframes_lazyload = [];
+
+		if ( rocket_avada_maybe_disable_lazyload() ) {
+			$disable_images_lazyload[] = __( 'Avada', 'rocket' );
+		}
+
 		if ( rocket_maybe_disable_lazyload() ) {
-			$rocket_maybe_disable_lazyload_plugins[] = __( 'Autoptimize', 'rocket' );
+			$disable_images_lazyload[] = __( 'Autoptimize', 'rocket' );
 		}
 
 		/**
-		 * Lazyload Helper filter which disables WPR lazyload functionality
+		 * Lazyload Helper filter which disables WPR lazyload functionality for images.
 		 *
 		 * @since  3.4.2
 		 * @author Soponar Cristina
 		 *
-		 * @param array Will return the array with all plugin names which should disable LazyLoad
+		 * @param array $disable_images_lazyload Will return the array with all plugin names which should disable LazyLoad
+		 */
+		$disable_images_lazyload = apply_filters( 'rocket_maybe_disable_lazyload_helper', $disable_images_lazyload );
+		$disable_images_lazyload = array_unique( array_filter( (array) $disable_images_lazyload ) );
+		$disable_images_lazyload = array_map( 'sprintf', array_fill( 0, count( $disable_images_lazyload ), '<strong>%s</strong>' ), $disable_images_lazyload );
+
+		/**
+		 * Lazyload Helper filter which disables WPR lazyload functionality for iframes.
+		 *
+		 * @since 3.5.5
+		 *
+		 * @param array $disable_iframes_lazyload Will return the array with all plugin names which should disable LazyLoad
 		 */
-		$rocket_maybe_disable_lazyload_plugins = apply_filters( 'rocket_maybe_disable_lazyload_helper', $rocket_maybe_disable_lazyload_plugins );
-		$rocket_maybe_disable_lazyload_plugins = wp_sprintf_l( '%l', $rocket_maybe_disable_lazyload_plugins );
+		$disable_iframes_lazyload = apply_filters( 'rocket_maybe_disable_iframes_lazyload_helper', $disable_iframes_lazyload );
+		$disable_iframes_lazyload = array_unique( array_filter( (array) $disable_iframes_lazyload ) );
+		$disable_iframes_lazyload = array_map( 'sprintf', array_fill( 0, count( $disable_iframes_lazyload ), '<strong>%s</strong>' ), $disable_iframes_lazyload );
+
+		$disable_lazyload = array_merge( $disable_images_lazyload, $disable_iframes_lazyload );
+		$disable_lazyload = array_unique( $disable_lazyload );
+
+		$disable_lazyload         = wp_sprintf_l( '%l', $disable_lazyload );
+		$disable_images_lazyload  = wp_sprintf_l( '%l', $disable_images_lazyload );
+		$disable_iframes_lazyload = wp_sprintf_l( '%l', $disable_iframes_lazyload );
```

Comment:
Why do we need to run both of these through `wp_sprintf_l()`. I don't see where we are rendering the values in each below. What am I missing?

Reply:
`$disable_iframes_lazyload` is not anymore indeed. `$disable_images_lazyload` is used [here](https://github.com/wp-media/wp-rocket/pull/2417/files#diff-e277836ca30970aac97804f2ca6ffbaeR1002).